### DISCUSSION TITLE
只能在linux系统下训练吗

### DISCUSSION THREAD

**Discussion by zyh0904** [STVIR/pysot#348] [General] [answered]
win10系统  不支持pytorch分布式  是这样吗？

**Comment by FrogMegane** [comment, marked as answer]
win10不支援pytorch distributed，不過我目前把dist相關的code都註解掉後可以使用單GPU在windows10下訓練

**Reply by DDuranin** [reply]
您好请问下您实现了pysot的单卡训练吗？如果可以，能麻烦您发我一份代码吗？十分感谢

**Comment by zyh0904** [comment]
> win10不支援pytorch distributed，不過我目前把dist相關的code都註解掉後可以使用單GPU在windows10下訓練

请问您是直接注释 什么代码都没加吗？ 我这边总是提示 rank 等变量出现问题，手动置为0，也总是提示不支持分布式训练。

**Comment by zyh0904** [comment]
> win10不支援pytorch distributed，不過我目前把dist相關的code都註解掉後可以使用單GPU在windows10下訓練

同学方便留个联系方式吗？最近因为疫情原因，只能在家做毕设了，有些问题还想请教您。我邮箱614295948@qq.com。可以直接加我QQ。谢谢啦~

**Comment by ZhiyuanChen** [comment]
> > win10不支援pytorch distributed，不過我目前把dist相關的code都註解掉後可以使用單GPU在windows10下訓練
> 
> 请问您是直接注释 什么代码都没加吗？ 我这边总是提示 rank 等变量出现问题，手动置为0，也总是提示不支持分布式训练。

你需要注释掉所有分布式训练的代码
